.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+**Environment information**
+* Mquery version:
+* Installation method:
+  - [ ] Generic docker-compose
+  - [ ] Dev docker-compose
+  - [ ] Native (from source)
+  - [ ] Other (please explain)
+
+**Reproduction Steps**
+
+Please explain how can we reproduce your problem.
```

Comment:
Seems like this line isn't commented `<!-- .... -->` while the others are. Any particular reason?